cmd/podman/pod_create.go

```diff
@@ -58,6 +58,7 @@ func init() {
 	// Remember to do this safely by checking len, etc.
 
 	flags.StringVar(&podCreateCommand.CgroupParent, "cgroup-parent", "", "Set parent cgroup for the pod")
+	flags.BoolVarP(&podCreateCommand.PinNamespaces, "pin-namespaces", "i", false, "Pin namespaces instead of using an infra container")
```

Comment:
for now, the "instead of using an infra container" isn't quite correct. I would want to update the description once we can actually drop the infra container

Reply:
oh I see that's the eventual intention of this PR, nevermind :smile: